sdks/php/src/Api/AccountApi.php

```diff
@@ -142,13 +153,13 @@ public function getResponse()
      *
      * Create Account
      *
-     * @param Model\AccountCreateRequest $account_create_request account_create_request (required)
+     * @param  Model\AccountCreateRequest $account_create_request account_create_request (required)
      *
-     * @return Model\AccountCreateResponse
      * @throws ApiException on non-2xx response or if the response body is not in the expected format
-     * @throws InvalidArgumentException
+     * @throws \InvalidArgumentException
+     * @return Model\AccountCreateResponse
      */
-    public function accountCreate(Model\AccountCreateRequest $account_create_request)
+    public function accountCreate($account_create_request)
```

Comment:
If you revert your changes to `sdks/php/.php-cs-fixer.dist.php` you'll see a massive reduction in the diff.